FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -27,6 +27,43 @@ namespace FEXCore::IR {
 class Pass;
 class PassManager;
 
+enum class MemoryAccessType {
+  // Choose TSO or Non-TSO depending on access type
+  ACCESS_DEFAULT,
```

Comment:
No need to use an `ACCESS_` prefix since it's already an enum class.

Reply:
Amended